backend/src/main/resources/liquibase/db-changelog-1.0.xml

```diff
@@ -518,6 +518,17 @@
         </update>
     </changeSet>
 
+    <changeSet id="add-repository-description-column" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="repository" columnName="description"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="repository">
+            <column name="description" type="TEXT"/>
```

Comment:
will it have default value which will cause NPE in toProto? Please check

Reply:
no, it will not give an NPE because when we are creating repository it will set `""` into the field at the constructor. so it will never null.

Reply:
I mean repositories that was already created with the old constructor (that didn't have description field). What value for description will it have?

Reply:
Ok got it your point. updating liquebase

Reply:
updated